currency/currency.go

```
@@ -168,3 +353,26 @@ func Round(currencyCode string, amount float64) (roundedAmount float64, err erro
 	}
 	return
 }
+
+// Equal returns true if two amounts are equal
+func Equal(currencyCode string, amount1 float64, amount2 float64) bool {
+	if !IsISO4217(currencyCode) {
+		return false
+	}
+	return Amount(currencyCode, amount1) == Amount(currencyCode, amount2)
+}
+
+// Amount returns a amount in a currency, fixed to the currency factor's decimal places
+func Amount(currencyCode string, amount float64) float64 {
+	if !IsISO4217(currencyCode) {
+		return 0
+	}
+	amount, _ = Round(currencyCode, amount)
+	return amount
+}
```

`if !IsISO4217(currencyCode)` is already called in `Round`.

Maybe we can rename to `RoundIgnoreErr` since the only difference with round is we ignored the error?

---

updated @aaron-wego